client/components/App.js

```diff
@@ -22,4 +28,19 @@ var App = React.createClass({
   },
 });
 
-module.exports = App;
+function mapStateToProps(state) {
+  return {
+    count: state,
+  };
+}
+
+function mapDispatchToProps(dispatch) {
+  return {
+    dispatch: (action) => dispatch(action)
+  };
+}
+
+module.exports = connect(
```

Comment:
This can be simplified to

```js
module.exports = connect(state => state)(App);
```

I'd possibly be tempted to structure this like a decorator

```js
var App = connect(state => state, React.createClass({
```